src/Region.ts

```diff
@@ -20,4 +6,5 @@ export enum RegionEnum {
  *
  * @public
  */
-export type Region = EnumOrString<RegionEnum>;
+
+export type Region = 'us' | 'eu';
```

Comment:
Not sure how important this is, but we should probably be consistent regarding whether we want this to be upper or lowercase. Right now, Region is always lowercase, but VersionLabel is all uppercase. Do we care about it being case-sensitive in the first place?

Reply:
> Do we care about it being case-sensitive in the first place?

While we could allow both, I feel like it may be better to pick one and stick to it. 

I've updated region to be capitalized, and force it lowercase when generating the request url.

Reply:
Perhaps we should check with Chris first about this.

Reply:
Discussed with Chris offline, he said we should allow both for Env and Region and I've updated the code to reflect this. For VersionLabel, since it's related to search I don't want to allow lowercase just in case this is not compatible with what they use it for, in addition to search not actually using the SDK atm.